mmtrack/models/sot/siamrpn.py

```diff
@@ -245,6 +247,138 @@ def track(self, img, bbox, z_feat, avg_channel):
         best_bbox = self._bbox_clip(best_bbox, img.shape[2], img.shape[3])
         return best_score, best_bbox
 
+    def _get_axis_aligned_bbox(self, quadrilateral):
+        """Convert polygon to axis aligned box in [cx, cy, w, h] format.
+
+        Args:
+            quadrilateral (Tensor): the coordinates of points in
+                [x1, y1, x2, y2, x3, y3, x4, y4] format. Specially, if the
+                shape is rectangle, the coordinate is in
+                [tl_x, tl_y, br_x, br_y] format.
+        Returns:
+            Tensor: in [cx, cy, w, h] format.
+        """
+        length = len(quadrilateral)
+        if length == 8:
+            cx = torch.mean(quadrilateral[0::2])
+            cy = torch.mean(quadrilateral[1::2])
+            x1 = torch.min(quadrilateral[0::2])
+            x2 = torch.max(quadrilateral[0::2])
+            y1 = torch.min(quadrilateral[1::2])
+            y2 = torch.max(quadrilateral[1::2])
+            A1 = torch.norm(quadrilateral[0:2] - quadrilateral[2:4]) * \
+                torch.norm(quadrilateral[2:4] - quadrilateral[4:6])
+            A2 = (x2 - x1) * (y2 - y1)
+            s = torch.sqrt(A1 / A2)
+            w = s * (x2 - x1)
+            h = s * (y2 - y1)
+            bbox = torch.stack((cx, cy, w, h))
+        elif length == 4:
+            bbox = bbox_xyxy_to_cxcywh(quadrilateral)
+        else:
+            NotImplementedError(f'The length of quadrilateral: {length} is \
+                 not supported')
+        return bbox
+
+    def _bbox_cxcywh_to_xywh(self, bbox):
+        """Convert bbox format from [cx, cy, w, h] to [x, y, w, h]"""
+        assert len(bbox) == 4
+        bbox[0] = max(bbox[0] - bbox[2] / 2, 0)
+        bbox[1] = max(bbox[1] - bbox[3] / 2, 0)
+        return bbox
+
+    def _bbox_xyxy_to_xywh(self, bbox):
+        """Convert bbox format from [x1, y1, x2, y2] to [x, y, w, h]"""
+        assert len(bbox) == 4
+        bbox[2] = max(bbox[2] - bbox[0], 0)
+        bbox[3] = max(bbox[3] - bbox[1], 0)
+        return bbox
+
+    def simple_test_vot(self, img, frame_id, gt_bboxes, image_shape=None):
```

Comment:
we need add the config of vot in the `test_forword/test_sot.py` of unittest in order to test `simple_test_vot`